stix_shifter_modules/nozomi/stix_translation/query_constructor.py

```diff
@@ -0,0 +1,549 @@
+            if value not in supported_values:
+                raise NotImplementedError(f'Unsupported ENUM values provided. {mapped_fields_array[0]} possible '
+                                          f"supported enum values are '{', '.join(supported_values)}'")
+            if comparator not in (ComparisonComparators.Equal, ComparisonComparators.NotEqual):
+                raise NotImplementedError('threat_name fields supports only for Equal/Not Equal operator')
+
+        if mapped_field_type == "properties" and \
+                comparator not in (ComparisonComparators.Equal, ComparisonComparators.NotEqual,
+                                   ComparisonComparators.Like):
+            raise NotImplementedError('Properties field supports only for Equal/Not Equal/Like operator')
+
+        if mapped_field_type == "epoch":
+            value = self._format_datetime(value)
+
+        if comparator == ComparisonComparators.IsSubSet and mapped_field_type != 'subset':
+            raise NotImplementedError(f'ISSUBSET operator allows only subset supported fields '
+                                      f'{self.config_map["subset_supported_fields"]}')
+
+        if expression.negated and comparator in (ComparisonComparators.GreaterThan, ComparisonComparators.LessThan,
+                                                 ComparisonComparators.GreaterThanOrEqual,
+                                                 ComparisonComparators.LessThanOrEqual,
+                                                 ComparisonComparators.IsSubSet):
+            raise NotImplementedError('Nozomi is not supported for NOT <, NOT >, NOT <=, NOT >=, NOT ISSUBSET'
+                                      ' operators')
+
+        if mapped_field_type == "bytes" and comparator in (ComparisonComparators.Equal, ComparisonComparators.NotEqual,
+                                                           ComparisonComparators.In):
+            value = f"{value} bytes"
+
+        return value
+
+    def _lookup_comparison_operator(self, expression_operator) -> str:
+        """
+        lookup operators support in nozomi
+        :param expression_operator:object
+        :return str
+        """
+        if str(expression_operator) not in self.comparator_lookup:
+            raise NotImplementedError(
+                f'Comparison operator {expression_operator.name} unsupported for nozomi connector')
+
+        return self.comparator_lookup[str(expression_operator)]
+
+    def _eval_comparison_value(self, expression, mapped_field_type, mapped_fields_array) -> str:
+        """
+        Function for parsing comparison expression value
+        :param expression: expression object
+        :param mapped_field_type: str
+        :param mapped_fields_array: list object
+        :return: formatted expression value
+        """
+        if expression.comparator == ComparisonComparators.Like:
```

Comment:
This can be done with the GreaterThan, LessThan, etc if statement right? It's doing the same thing under both.

Reply:
Updated as per the suggestion.